homeassistant/components/date/strings.json

```
@@ -0,0 +1,3 @@
+{
+  "title": "Date"
```

Add translations strings for the entity component (this currently does assume we keep the state attributes around).


```suggestion
  "title": "Date",
  "entity_component": {
    "_": {
      "name": "[%key:component::date::title%]",
      "state_attributes": {
        "day": {
          "name": "Day"
        },
        "month": {
          "name": "Month"
        },
        "year": {
          "name": "Year"
        }
      }
    }
  }
```